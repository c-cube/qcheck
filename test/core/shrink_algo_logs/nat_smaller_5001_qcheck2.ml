open QCheck2

let nats_smaller_5001 =
  Test.make ~name:"nat < 5001"
    Gen.nat (Log.shrinks_int @@ fun n -> n < 5001)

let rand = Random.State.make [|1234|];;
try Test.check_exn ~rand nats_smaller_5001 with Test.Test_fail _ -> ()
