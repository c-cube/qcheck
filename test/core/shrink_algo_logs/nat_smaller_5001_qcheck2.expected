holds 250
holds 7
holds 97
holds 174
holds 9
holds 2
holds 1
holds 85
holds 544
holds 57
holds 9
holds 68
holds 560
holds 0
holds 4
holds 44
holds 8
holds 206
holds 4
holds 590
holds 4
holds 75
holds 8
holds 8
holds 825
holds 4
fails 5624
holds 0
holds 2812
holds 4218
holds 4921
fails 5273
holds 0
holds 2636
holds 3954
holds 4613
holds 4943
fails 5108
holds 0
holds 2554
holds 3831
holds 4470
holds 4789
holds 4949
fails 5029
holds 0
holds 2514
holds 3771
holds 4400
holds 4714
holds 4871
holds 4950
holds 4989
fails 5009
holds 0
holds 2504
holds 3756
holds 4382
holds 4695
holds 4852
holds 4930
holds 4969
holds 4989
holds 4999
fails 5004
holds 0
holds 2502
holds 3753
holds 4379
holds 4692
holds 4848
holds 4926
holds 4965
holds 4985
holds 4995
holds 5000
fails 5002
holds 0
holds 2501
holds 3752
holds 4377
holds 4690
holds 4846
holds 4924
holds 4963
holds 4983
holds 4993
holds 4998
holds 5000
fails 5001
holds 0
holds 2500
holds 3750
holds 4375
holds 4688
holds 4844
holds 4922
holds 4961
holds 4981
holds 4991
holds 4996
holds 4998
holds 4999
holds 5000
